selfdrive/car/honda/interface.py

```diff
@@ -273,6 +273,18 @@ def get_params(candidate, fingerprint):
       ret.longitudinalTuning.kiBP = [0., 35.]
       ret.longitudinalTuning.kiV = [0.18, 0.12]
 
+    elif candidate == CAR.FIT:
+      stop_and_go = False
+      ret.mass = 2644. * CV.LB_TO_KG + std_cargo
+      ret.wheelbase = 2.53
+      ret.centerToFront = ret.wheelbase * 0.39
+      ret.steerRatio = 13.06
+      ret.steerKpV, ret.steerKiV = [[0.8], [0.24]]
```

Comment:
```suggestion
      tire_stiffness_factor = 0.444 # not optimized yet
      ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.25], [0.06]]
```
Does tire_stiffness_factor is essential? How do we tune it?
Needed lat kpv and kiv reduced to get rid of steering vibration

Reply:
Tire stiffness affects oversteer/understeer. It should be set here even if its set to factor of 1.